packages/forma-36-react-components/.size-limit

```diff
@@ -12,7 +12,7 @@
     {
       "path": "dist/styles.css",
       "name": "gzipped css",
-      "limit": "75 KB",
+      "limit": "85 KB",
```

Comment:
Is the animation actually increasing the library 10kb gzipped?

Reply:
It's increasing in 5.something KB

I can make the limit `80.5 KB`

Reply:
Is it still too much 5kb? What can we do to make it lighter? 🤔

Reply:
It must compress very poorly. I'm not sure why.